### ISSUE TITLE
When sending "where" command twice, "where:stop" and "stop" command don't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#348]
```
===  IMPORTANT - READ THIS!  ===
An app log will help us greatly to analyze the problem. So ALWAYS generate
an AppLog, even if it does not show anything (which will tell use that we
have to improve the log part ;-) ).
http://code.google.com/p/gtalksms/wiki/GenerateAppLog
You can remove this message from the issue report if you have read it.
================================

What steps will reproduce the problem?
1.
2.
3.

What is the expected output?


What do you see instead?


Additional informations:
Phone model: <enter phone model>

```

Original issue reported on code.google.com by `yzsr2013...@gmail.com` on 14 Jan 2014 at 3:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 22 Feb 2014 at 1:51
- Changed title: **When sending "where" command twice, "where:stop" and "stop" command don't work**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 92e4f66b9ec7.
```

Original comment by `yako...@gmail.com` on 22 Feb 2014 at 2:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4.5
```

Original comment by `Florent....@gmail.com` on 15 Aug 2014 at 7:18
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
